it/uk/gov/hmrc/emailverification/JourneyWireMockSpec.scala

```diff
@@ -83,6 +83,32 @@ class JourneyWireMockSpec extends BaseISpec with Injecting {
           )
         }
       }
+
+      "the email should contain the Welsh service name when all fields are present" in new Setup {
+        expectEmailToSendSuccessfully()
+        val labels = Json.obj(
+          "cy" -> Json.obj(
+            "pageTitle" -> "Page Title.cy",
+            "userFacingServiceName" -> "Team Name.cy"
+          ),
+          "en" -> Json.obj(
+            "pageTitle" -> "Page Title.en",
+            "userFacingServiceName" -> "Team Name.en"
+          ),
+        )
+        val response = await(resourceRequest("/email-verification/verify-email").post(verifyEmailWithLabelsPayload(emailAddress, labels)))
+
+        val uuidRegex = "[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}"
+
+        response.status shouldBe CREATED
+        (response.json \ "redirectUri").as[String] should fullyMatch regex s"/email-verification/journey/$uuidRegex/passcode\\?continueUrl=$continueUrl&origin=$origin&service=Team Name.cy"
```

Comment:
As discussed on a call, please can you remove (in tests & code) the end part of the redirectUri "&service=Team Name.cy" as that shouldn't be there.

Also, check the FE isn't using it - I suspect not.

The FE can determine everything it needs from the BE by using the journeyId to look up anything it requires.  Thanks.